docs/analyzer-rules/AZFW0009.md

```diff
@@ -0,0 +1,23 @@
+# AZFW0009: Invalid EventHubs Trigger
+
+| | Value |
+|-|-|
+| **Rule ID** |AZFW0009|
+| **Category** |[FunctionMetadataGeneration]|
+| **Severity** |Error|
+
+## Cause
+
+This rule is triggered when the EventHubs trigger of an Azure Function is invalid.
+
+## Rule description
+
+EventHubs triggers must correctly declare a compatible "IsBatched" value and parameter input type. For example, for EventHubs triggers where `IsBatched = true`, the input parameter type must be an iterable collection.
```

Comment:
_note: The default value of `IsBatched` is true. So if you are not explicitly providing a value, the parameter type should be a collection type._